src/mudlet.pro

```diff
@@ -860,9 +869,16 @@ HEADERS += \
     widechar_width.h \
     ../3rdparty/discord/rpc/include/discord_register.h \
     ../3rdparty/discord/rpc/include/discord_rpc.h
-    
+
 macx {
-    HEADERS += ../3rdparty/sentry-native/include/sentry.h
+    contains( DEFINES, INCLUDE_SENTRY ) {
+        # sentry-native is needed for MacOS builds with crash-reporting
+        exists("$${PWD}/../3rdparty/sentry-native/sentry-config.cmake.in") {
```

Comment:
That is a CMake file, but this is a qmake project file - is there a more "neutral" file we can use for this. I know it is a bit incidental but it would appear cleaner if they don't concern themselves with each other's files. :man_shrugging:

Reply:
I'm not able to think of a more "neutral" file for managing sentry specific checks. Currently, most of the checks are written taking inspiration from validations that we have for other 3rd party libraries 😅